trl/trainer/grpo_trainer.py

```diff
@@ -1239,10 +1241,11 @@ def _generate_and_score_completions(
             self._metrics[mode][f"rewards/{reward_func_name}/std"].append(std_rewards)
         self._metrics[mode]["reward"].append(mean_grouped_rewards.mean().item())
         self._metrics[mode]["reward_std"].append(std_grouped_rewards.mean().item())
-
+        self._metrics[mode]["frac_zero_reward_std"].append(frac_zero_std.float().mean().item())
```

Comment:
To avoid confusion with users thinking this refers to "zero reward" instead of "rewards with zero std dev", let's rename to this:
```suggestion
        self._metrics[mode]["frac_reward_zero_std"].append(frac_zero_std.float().mean().item())
```